karma.conf.js

```diff
@@ -25,6 +25,7 @@ module.exports = function(config) {
     coverageReporter: {
       reporters: [
         { type: 'html', subdir: 'report-html' },
+        { type: 'json', subdir: 'report-json' },
```

Comment:
To get istanbul to output `coverage.json`. I was trying to increase the coverage a bit, and that output seemed to be better at pointing out the specific lines without coverage.